biz.aQute.bnd.reporter/src/biz/aQute/bnd/reporter/plugins/serializer/JsonReportSerializerPlugin.java

```diff
@@ -17,7 +17,6 @@
 	private final Encoder			enc		= new JSONCodec().setIgnorenull(true)
 		.enc()
 		.keepOpen()
-		.indent("  ")
```

Comment:
XmlConverterPluginTest > testXmlConvertion did not test against an indented json.
https://github.com/bndtools/bnd/blob/ba0bd37c2c33afe4ccfd8660b1a37b1c1ac52eaa/biz.aQute.bnd.reporter/testresources/xmlConverter/xmlJsonResult.json